TODO.md

```diff
@@ -68,3 +68,5 @@
   every read/write range command.
 
 * Gossip should be changed to send only protobufs.
+* persist Gossip and the kv.leaderCache to make sure that restarting
+  a node is harmless.
```

Comment:
I'm not sure if either of these is desirable either from a semantic or performance perspective. Shouldn't the leader cache fill up quickly upon restart? And ditto for the gossip info?